CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
 ```
 
-> **Note**: certain versions of npm (5-8) and Node.js (10-16) are required to run certain scripts.
-
```

Comment:
The hover state looks a bit odd since there's no padding on the left: 
![image](https://github.com/ionic-team/ionic-docs/assets/2721089/911acba1-f8a7-458d-b564-be89c5014580)


This looks like a new behavior in the latest version of Docusuarus